packages/text-block/src/TextBlock.spec.ct.tsx

```diff
@@ -22,32 +22,18 @@ describe('Text Block', () => {
         cy.get(RichTextEditor).find('[contenteditable=true]').should('not.exist');
     });
 
-    it('should be able input to two text blocks when in edit mode', () => {
```

Comment:
not needed anymore?

Reply:
not really, as the behaviour of two or more columns is now just solved with tailwind classes. And this is tested in the last test. Do you think it needs to be tested more detailed?